### ISSUE TITLE
Is it possible to access the localized string programmatically

### ISSUE THREAD

**Issue by ricsantos** [kishikawakatsumi/applelocalization-web#22]
Firstly, thanks @kishikawakatsumi for the awesome resource, its very well done.

I'm wondering, is there a way to access the strings at runtime? 

For example:
`ACTIVE_ENERGY_TITLE_EMBEDDED`
"Active Energy"
[HealthKit.framework](https://applelocalization.com/?b=HealthKit.framework)
Localizable-DataTypes.strings

I am trying to get it like so, but it's not working, either because I'm missing something

    let bundle = Bundle(for: HKWorkout.self)
    let string = NSLocalizedString("ACTIVE_ENERGY_TITLE_EMBEDDED", tableName: nil, bundle: bundle, value: "", comment: "")

**Comment by kishikawakatsumi** [comment]
@ricsantos 

`ACTIVE_ENERGY_TITLE_EMBEDDED` key is defined in `Localizable-DataTypes.strings`

<img width="1615" alt="Screen Shot 2022-08-18 at 2 39 45" src="https://user-images.githubusercontent.com/40610/185206284-dec56190-847e-4ab9-be3b-614859702075.png">

So you need to specify `tableNeme` `"Localizable-DataTypes"`.

```
let bundle = Bundle(for: HKWorkout.self)
let string = NSLocalizedString("ACTIVE_ENERGY_TITLE_EMBEDDED", tableName: "Localizable-DataTypes", bundle: bundle, value: "", comment: "")
```

The above code should return `"Active Energy"` string.
I only check on a simulator, but on a device should work as same.

**Comment by ricsantos** [comment]
That makes perfect sense! Thank you very much @kishikawakatsumi, I appreciate your help.

I actually have a follow up question, I am also trying to get the strings for the `HKWorkoutActivityType` enum, eg "Basketball"

I notice that `HKWorkoutActivityTypeBasketball` it's in `FitnessUI.framework` which is private, so I guess I can't use that.

I notice also that it's in `HealthUI.framework` but I can't see that, only `HealthKitUI.framework`, maybe it's also private?

**Comment by kishikawakatsumi** [comment]
Yes, `HealthUI.framework` is a private framework. If you'd like to access the bundle, you can use `Bundle(path:)` initializer. Like the following:

```
let bundle = Bundle(path: "/System/Library/PrivateFrameworks/HealthUI.framework")!
let string = NSLocalizedString("HKWorkoutActivityTypeBasketball", tableName: "HealthUI-Localizable", bundle: bundle, value: "", comment: "")
```

^ this should show `"Baseball"` string. It should work only on a device.

**Comment by ricsantos** [comment]
Oh yeah it works, you're the best. Thanks again!
